keras_nlp/layers/modeling/rotary_embedding_test.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+from keras_nlp.layers.modeling.rotary_embedding import RotaryEmbedding
+from keras_nlp.tests.test_case import TestCase
+
+
+class RotaryEmbeddingTest(TestCase):
```

Comment:
We should definitely fill out the test case now that we are exposing this standalone. Maybe look of the `SinePositionEncoding` layer tests as a start.

Reply:
Yup, added tests from SinePositionEncoding layer, they all pass now for all backends